crates/bevy_pbr/src/wireframe.rs

```diff
@@ -41,158 +31,101 @@ impl Plugin for WireframePlugin {
         app.register_type::<Wireframe>()
             .register_type::<WireframeConfig>()
             .init_resource::<WireframeConfig>()
-            .add_plugins((
-                ExtractResourcePlugin::<WireframeConfig>::default(),
-                ExtractComponentPlugin::<Wireframe>::default(),
-            ));
-
-        if let Ok(render_app) = app.get_sub_app_mut(RenderApp) {
-            render_app
-                .add_render_command::<Opaque3d, DrawWireframes>()
-                .init_resource::<SpecializedMeshPipelines<WireframePipeline>>()
-                .add_systems(Render, queue_wireframes.in_set(RenderSet::Queue));
-        }
-    }
-
-    fn finish(&self, app: &mut bevy_app::App) {
-        if let Ok(render_app) = app.get_sub_app_mut(RenderApp) {
-            render_app.init_resource::<WireframePipeline>();
-        }
+            .add_plugins(MaterialPlugin::<WireframeMaterial>::default())
+            .add_systems(Update, apply_global)
+            .add_systems(Update, apply_material);
```

Comment:
This plus `cargo fmt` as the suggestion was hand-written.
```suggestion
            .add_systems(Update, (
                apply_global,
                apply_material,
            ));
```